Stanford/JumpstartAcademic/Install/Menu/MenuRedirects.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * @file
+ * Abstract Task Class.
+ */
+
+namespace Stanford\JumpstartAcademic\Install\Menu;
+/**
+ *
+ */
+class MenuRedirects extends \AbstractInstallTask {
+
+  /**
+   * Set the site name.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+    // Create redirects.
+    $redirects = array(
+      'academics' => 'academics/academics-overview',
+      'people' => 'people/faculty',
+      'news' => 'news/recent-news',
+      'events' => 'events/upcoming-events',
+      'about' => 'about/about-us',
+    );
+
+    foreach ($redirects as $source => $dest) {
```

Comment:
This can be reworked to use `Stanford\Utility\Install\CreateRedirects` once #10 is merged.